src/cmd/run_canary_mode.rs

```diff
@@ -0,0 +1,50 @@
+use async_trait::async_trait;
```

Comment:
Definitely rename these files to follow the module layout conventions a little more closely.

1. Create a directory named `run`
2. Put the `Mode` trait into `run/mod.rs`
3. Export both `ForceMode` and `CanaryMode` from `run/mod.rs`.
4. Create `run/canary.rs` and declare `CanaryMode` in that file.
5. 4. Create `run/force.rs` and declare `ForceMode` in that file.